### PR TITLE
Add details to Exception when data_xciever.connect() fails.

### PR DESCRIPTION
**Pull request by ghedsouza**

### CODE REVIEW

**Comment by ghedsouza** [comment]
https://github.com/spotify/snakebite/issues/154

**Comment by Tarrasch** [comment]
@ravwojdyla, you should fix the general build errors, see https://github.com/spotify/luigi/commit/14819e885d30c024e6778013534a2411e0f9d035

**Comment by Tarrasch** [comment]
Thanks for the patch @ghedsouza. While you're at it, why don't you pick a more narrow Exception type over just `Exception`?

**Comment by ghedsouza** [comment]
@Tarrasch Okay, thanks for the feedback. I see two options here:

1. Add a `connectionFailureException` to https://github.com/ghedsouza/snakebite/blob/master/snakebite/errors.py

2. Use https://docs.python.org/2/library/exceptions.html#exceptions.RuntimeError

I'm fine with either one.

**Comment by Tarrasch** [comment]
I think 1) is the best. I'm also fine with either. :)

File: snakebite/errors.py [hunk 1]
```diff
@@ -14,29 +14,29 @@
 # the License.
 
 
-class FileNotFoundException(Exception):
```
Comment (ghedsouza) [inline snakebite/errors.py]: Alphabetically sorted all the exceptions here.

**Comment by ghedsouza** [comment]
@Tarrasch Updated.

**Comment by Tarrasch** [comment]
I'll let @ravwojdyla clcik the merge button

**Comment by ravwojdyla** [comment]
Thanks:
* @ghedsouza for PR - looks good.
* @Tarrasch for help! Addressing your comments via #156 

@ghedsouza let's fix test/build via #156, when it's solved I will trigger tests again and if everything is fine we will merge. Will get back soon.

**Comment by ghedsouza** [comment]
@ravwojdyla Tests passing now. Almost don't believe it :p

**Comment by wouterdebie** [comment]
@ravwojdyla ping!

**Comment by ghedsouza** [comment]
@Tarrasch or @ravwojdyla planning to merge this?

**Comment by Tarrasch** [comment]
Sorry I can't (shouldn't) merge this. I'll IRL-ping @ravwojdyla if I get the chance though! :)

**Pull request merged**